src/modules/rppt_tensor_augmentations.cpp

```diff
@@ -774,3 +774,86 @@ rppt_color_cast_host(RppPtr_t srcPtr,
 
     return RPP_SUCCESS;
 }
+
+/******************** box_filter ********************/
+
+RppStatus
+rppt_box_filter_gpu(RppPtr_t srcPtr,
+                    RpptDescPtr srcDescPtr,
+                    RppPtr_t dstPtr,
+                    RpptDescPtr dstDescPtr,
+                    Rpp32u kernelSize,
+                    RpptROIPtr roiTensorPtrSrc,
+                    RpptRoiType roiType,
+                    rppHandle_t rppHandle)
+{
+#ifdef OCL_COMPILE
+
```

Comment:
Is this left intentional?

Reply:
In all the new tensor functions we are developing, I have left OCL blank in case there comes a need to add an ocl implementation later. If not, I could remove it for these new set of tensor functions.